web/migrations/0052_encrypt_all_pii.py

```
@@ -0,0 +1,11 @@
+# Generated by Django 5.1.6 on 2025-04-03 10:56
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("web", "0051_teamgoalmember_completion_image_and_more"),
+    ]
+
+    operations = []
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)